### PR TITLE
highlights(php): fix and add missing highlights

### PR DESCRIPTION
**Pull request by mikehaertl**
@theHamsta As mentioned in #2207 here are a couple of additions for PHP

 * **@type**: added class and interface names in declarations, `use` statements and static method calls
 * **@parameter** parameters in method and function declarations
 * **@const** constant declarations and accesses
 * **@namespace** namespace names
 * **@conditional** added ternary operator `? :`
 * **@string** added backtick expressions
 * **@string.escape** escape expressions like `\n` in strings
 * **@label** goto labels
 
Also added some keywords and operators and moved some things e.g. the `.` operator to **@operators**

### CODE REVIEW

**Comment by mikehaertl** [comment]
@infectos Maybe you also want to help test this? Suggestions welcome.

File: queries/php/highlights.scm [hunk 1]
```diff
@@ -173,14 +214,17 @@
 [
   "="
 
+  "."
```
Comment (theHamsta) [inline queries/php/highlights.scm:217]: Why is this an operator? In other languages foo.bar is punctuation. If this needs a change it should be applied to all languages

**Review by theHamsta — COMMENTED** [review]
Looks good to me! 👍

File: queries/php/highlights.scm (hunk 1 above)
Comment (mikehaertl) [reply queries/php/highlights.scm:217]: Dot is the [string concatenation operator](https://www.php.net/manual/en/language.operators.string.php):
```php
<?php
$a = 'some ' . 'text';
```

In PHP object properties are always accessed with the arrow `->` not the `.`: 
```php
<?php
$a->someProp;
$a->someMethod();
```

Static props/methods use the `::` operator:
```php
<?php
$a = SomeClass::$prop;
$a = SomeClass::someMethod();
```

**Comment by mikehaertl** [comment]
@theHamsta I've become a big fan of testing lately. But the existing tests for other languages seem to be rather rudimentary. Are there plans to extend this? And does it make sense to add tests for PHP? IMO a test should cover pretty much all of the important features and is quite some work though. So in any case this would be something for later.

File: queries/php/highlights.scm (hunk 1 above)
Comment (theHamsta) [reply queries/php/highlights.scm:217]: Yeah, I had the suspicion that is was concatenation but didn't whether it was also object access

**Comment by theHamsta** [comment]
Feel free to add tests. The test facilities were only added very recently and after that only additions and changes received tests.

**Comment by mikehaertl** [comment]
@theHamsta Ok I may consider to add tests later in another PR.

**Review by theHamsta — APPROVED** [review]

**Pull request merged**